pkg/api/meta/restmapper.go

```diff
@@ -335,3 +351,13 @@ func (m MultiRESTMapper) AliasesForResource(alias string) (aliases []string, ok
 	}
 	return nil, false
 }
+
+// ResourceInGroup verifies if the group has the resource in it
+func (m MultiRESTMapper) ResourceInGroup(group, resource string) bool {
+	for _, t := range m {
```

Comment:
Can you not directly find a t of the right group? Seems odd to loop and call ResourceInGroup on every t.